pkg/hyper/helper.go

```diff
@@ -39,6 +40,8 @@ const (
 	fraktiAnnotationLabel = "io.kubernetes.frakti.annotations"
 )
 
+type Items []*kubeapi.PodSandbox
```

Comment:
nit: `Items` should be renamed to something more specific, e.g. `sandboxByCreated`